README.md

```diff
@@ -98,7 +98,14 @@ See all Kafka Connect [configuration parameters](https://docs.confluent.io/curre
 
 ## Connector-specific Parameters
 
-See `kafka-connect-datagen` [configuration parameters](https://github.com/confluentinc/kafka-connect-datagen/blob/master/src/main/java/io/confluent/kafka/connect/datagen/DatagenConnectorConfig.java) and their defaults.
+Parameter | Description | Default
+-|-|-
+kafka.topic | Topic to write to | 
```

Comment:
All parameter names should have backticks.  e.g. `kafka.topic` instead of kafka.topic

Reply:
Done